api/audit/tasks.py

```diff
@@ -48,11 +50,11 @@ def _create_feature_state_audit_log_for_change_request(
         feature_state.change_request.title,
     )
     AuditLog.objects.create(
-        related_object_id=feature_state.id,
-        related_object_type=RelatedObjectType.FEATURE_STATE.name,
-        environment=feature_state.environment,
-        project=feature_state.environment.project,
+        environment=feature_state._get_environment(),
+        project=feature_state._get_project(),
```

Comment:
They are only private because they were already private - I guess they were intended for use by the public `get_environment_and_project` (now `get_organisations_project_environment`). 
Previously this code accessed member variables directly, which seemed to be against DRY and introduce the possibility that it could pick different values to those used for audit logs that go via `_get_environment`.
I guess there are some options here:

- use `get_organisations_project_environment` public interface (like `create_audit_log_from_historical_record` in the same file does)
- make `_get_organisations/project/environment` into public interface

Reply:
I guess the downside of option 1 here is that we don't need the organisations (I don't think?). My preference in that case would be option 2 but if, for whatever reason, we do want the organisation here too, let's go with option 1.

Reply:
We don't need organisation so went with option 2